website/templates/emails/withdrawal_request_declined.html.mako

```diff
@@ -6,16 +6,18 @@
     <%!
         from website import settings
     %>
-        Dear ${requester.fullname},
-		<p>
         % if document_type == 'registration':
+            Dear ${contributor.fullname},
+		    <p>
             Your request to withdraw your registration <a href="${reviewable.absolute_url}">"${reviewable.title}"</a> from ${reviewable.provider.name} has been declined by the service moderators. The registration is still publicly available on ${reviewable.provider.name}.
 			<p>
             % if notify_comment:
                 The moderator has provided the following comment:<br>
                 ${comment}
             % endif
         % else:
+            Dear ${requester.fullname},
```

Comment:
This email is new for Registrations Moderation, so please go ahead and make the fix to `contributor` instead of `requester` here universal

Reply:
I did this because when this email is sent as a result of preprints, there is no `contributor` attribute in the context (See https://github.com/CenterForOpenScience/osf.io/blob/develop/osf/utils/machines.py#L289). My understanding is that preprints only have a requester in the context because because a preprint withdrawal request has a single creator who the email is sent to whereas for a registration, the withdrawal request email is sent to all contributors on the registration.

Reply:
Good catch!!